src/NexusMods.App.UI/WorkspaceSystem/Panel/PanelViewModel.cs

```diff
@@ -65,6 +65,11 @@ public PanelViewModel(IWorkspaceController workspaceController, PageFactoryContr
             .Bind(out _tabs)
             .Subscribe();
 
+        this.WhenAnyValue(vm => vm.SelectedTabId)
+            .Select(selectedTabId => Tabs.FirstOrDefault(tab => tab.Id == selectedTabId))
+            .WhereNotNull()
+            .SubscribeWithErrorLogging(selectedTab => SelectedTab = selectedTab);
```

Comment:
Not a fan of this. This isn't fixing the underlying issue.

Reply:
Which would be that it can still be null you mean? Or what problem are you referring to?